app/src/main/java/com/parking_system_kotlin/mvp/view/ParkingReservationView.kt

```diff
@@ -25,7 +28,32 @@ class ParkingReservationView(activity: Activity, private val binding: ActivityRe
         binding.editTextReservationActivityExit.setText(date)
     }
 
+    private fun showMessage(messageID: Int) {
+        context?.let {
+            Toast.makeText(it, messageID, Toast.LENGTH_LONG)
+                .show()
+        }
+    }
+
+    override fun showConfirmationMessage() {
+        showMessage(R.string.reservation_activity_text_confirmation_message)
+    }
+
+    override fun showErrorMessage() {
+        showMessage(R.string.reservation_activity_text_parking_lot_error_message)
+    }
+
     override fun closeScreen() {
         activity?.finish()
     }
+
+    override fun getDataForReservation(): Reservation {
+        return Reservation(
```

Comment:
the presenter is the one that should create the object, the view will only provide the info for that

Reply:
done